builtin/rebase.c

```diff
@@ -12,6 +12,11 @@
 #include "refs.h"
 #include "dir.h"
 #include "packfile.h"
+#include "lockfile.h"
+#include "unpack-trees.h"
```

Comment:
Likewise, I don't think that we need to include `unpack-trees.h` in `builtin/rebase.c`...